css/styles.css

```diff
@@ -10,22 +10,15 @@
 
 div.main-container {
   background-color: #f2ebe9;
+  height: 100vh;
 }
 
 header.navigation {
   background-color: #243a73;
   width: 100%;
-  position: sticky;
-  height: 70px;
+  height: 9%;
```

Comment:
I recommend sizing a header's height based on its content, not based on its parent's height.